docs/jwst/user_documentation/running_pipeline_python.rst

```diff
@@ -529,3 +529,163 @@ individual step parameter must be set when using this method, or else the coded
 defaults will be used, which may be inappropriate for the dataset being processed.
 
 See :ref:`call_examples` for more information.
+
+
+.. _multiprocessing:
+
+Multiprocessing
+===============
+
+Multiprocessing is supported to speed up certain computationally-intensive steps
+in the pipeline, including the :ref:`jump detection <jump_step>`,
+:ref:`ramp fitting <ramp_fitting_step>`, and
+:ref:`WFSS contamination correction <wfss_contam_step>` steps. The examples below show how
+multiprocessing can be enabled for these steps, as well as how to set up
+multiprocessing to simultaneously run the entire pipeline on multiple observations.
+
+Since the pipeline uses multiprocessing it is critical that any code using the pipeline adhere to the guidelines
+described in the `python multiprocessing documentation 
+<https://docs.python.org/3/library/multiprocessing.html#multiprocessing-programming>`_.
```

Comment:
```suggestion
Since the pipeline uses multiprocessing it is critical that any code using the pipeline adhere
to the guidelines described in the
`python multiprocessing documentation <https://docs.python.org/3/library/multiprocessing.html#multiprocessing-programming>`_.
```